functions/controllers/funkoPopController.js

```diff
@@ -42,24 +55,40 @@ const getFunkoPopName = async (req, res, next) => {
     const data = await funkoPops.get();
     const funkoArray = [];
     if (data.empty) {
-      res.status(404).send("No Funko Pop records found");
+      res.status(404).send("No Funko Pop records exsist");
     } else {
       data.forEach((doc) => {
+        console.log(doc.data());
         const funkoObj = new FunkoPop(doc.data().genre, doc.data().funkoData);
+        console.log(funkoObj);
         funkoArray.push(funkoObj);
       });
       console.log(funkoArray);
+      const matchArr = [];
       funkoArray.forEach((funko) => {
+        const genre = funko.genre;
         const funkoData = funko.funkoData;
-        const matches = funkoData.filter((data) =>
-          data.name.toLowerCase().includes(query)
-        );
-        console.log(matches);
+        console.log("funkoData: ", funkoData);
+        if (funkoData) {
+          if (funkoData.name) {
```

Comment:
we don't need 2 if checks here.  just if (funkoData?.name) or (funkoData && funkoData.name)

Reply:
not using this function anymore, have it just for reference but i will change it